docs/content/dagster-cloud/managing-deployments/alerts.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: "Dagster Cloud alerts | Dagster Docs"
+description: "Create and configure alerts in Dagster Cloud."
+
+platform_type: "cloud"
+feature: "alerting"
+---
+
+# Dagster Cloud alerts
+
+<ApplicableDagsterProduct />
+
+Using Dagster Cloud's built-in alerting system, you can stay in the loop on the events occurring in your deployment. Dagster Cloud alerts integrate with several popular services, allowing you to deliver notifications when and where they're most needed.
+
+---
+
+## How it works
+
+Alerts are created using **alert policies**. An alert policy defines [the conditions that will trigger an alert](#alert-policy-types) and [how the alert will be sent](#supported-notification-services). For example, you can create a policy that sends an alert to a Slack channel when any asset in a specific asset group fails to materialize.
+
+Alert policies are configured on a **per-deployment basis**. This means, for example, that asset alerts configured in a `prod` deployment are only applicable to assets in that deployment.
+
+### Alert policy types
+
+<table
+  className="table"
+  style={{
+    width: "100%",
+  }}
+>
+  <thead>
+    <tr>
+      <th
+        style={{
+          width: "25%",
+        }}
+      >
+        Policy type
+      </th>
+      <th>How it works</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <strong>Asset alert</strong>
+      </td>
+      <td>
+        Triggers on:
+        <ul>
+          <li>
+            <strong>Asset materialization</strong> - Failure or success
+          </li>
+          <li>
+            <strong>Asset check</strong> - Error, warn, passed, or failure to
+            execute
+          </li>
+          <li>
+            <strong>Asset freshness</strong> - Overdue
+          </li>
+        </ul>
+        An asset group or asset key can be provided, which limits notifications
+        to fire only if the asset group or asset key matches the materialized
+        asset. In the case of checks, notifications will only be sent if the
+        asset group/key to which the check is attached matches.
+        <br />
+        <br />
+        <strong>Note:</strong> Asset based alert policies are still experimental
```

Comment:
we can remove this https://dagsterlabs.slack.com/archives/C06FFA6CQ5N/p1710805946587069